translations/SECURITY.Japanese.md

```diff
@@ -0,0 +1,41 @@
+## 脆弱性の報告
+
+セキュリティ研究者と脆弱性を報告するユーザーに非常に感謝しています Hasura コミュニティ。 
+すべてのレポートは、一連のコミュニティボランティアによって徹底的に調査され、Hasura チーム。
+
+セキュリティの問題を報告するには、メールでお問い合わせください  [build@hasura.io](mailto:build@hasura.io) 必要なすべての情報を添付して、
+すべての詳細を記載します。
+
+### いつ脆弱性を報告する必要がありますか ?
+
+- あなたはあなたが潜在的なセキュリティ脆弱性を発見したと思う Hasura GraphQL Engine または関連コンポーネント。
+- 脆弱性がどのように影響するかわからない Hasura GraphQL Engine.
+- あなたは別のプロジェクトで脆弱性を発見したと思う Hasura GraphQL Engine オン (例えば. Heroku, Docker, 等).
+- 潜在的に危害を加える可能性のある他のセキュリティリスクを報告する場合 Hasura GraphQL Engine ユーザー。
+
+### 脆弱性を報告すべきではない場合?
+
+- チューニングの助けが必要 Hasura GraphQL Engine セキュリティのためのコンポーネント。
+- セキュリティ関連の更新を適用するのに助けが必要です。
+- 問題はセキュリティ関連ではありません。
+
+### セキュリティ脆弱性への対応
+
+- 各レポートは、プロジェクトのメンテナーとセキュリティチームによって3営業日以内に承認および分析されます。
+- レポーターは、各段階で更新され続けます 問題の 分析と解決 （トリアージ->修正->リリース）。
+
+## 公 開示 タイミング
+
+公開の日付はによって交渉されます Hasura 製品 保安 チーム そしてその バグ提出者。 私達 好む に 完全に 開示する その バグ できるだけ速やかに 
+ユーザー緩和策が利用可能になったら。 バグまたは修正がまだ完全に理解されていない場合、開示を遅らせることは合理的です、 
+ソリューションは十分にテストされていません、またはベンダーの調整のため。開示の時間枠は、即時（特に既に公開されている場合）から数週間です。
+報告から一般公開までの期間は、通常7日程度であると予想されます。その Hasura GraphQL Engine メンテナーとセキュリティチームは、
+開示日を設定する最終的な電話をかけます。
+
+（一部のセクション なっている 触発された そして 適合しました から 
+[https://github.com/kubernetes/website/blob/master/content/en/docs/reference/issues-
+security/security.md](https://github.com/kubernetes/website/blob/master/content/en/docs/reference/issues-security/security.md))
+
+## 翻訳
+このドキュメントは、次の翻訳版で入手できます:
+- [French :fr:](translations/SECURITY.french.md)
```

Comment:
Could you please add the English translation here?

Reply:
## Vulnerability report

We are very grateful to security researchers and users reporting vulnerabilities in the Hasura community.
All reports are thoroughly investigated by a series of community volunteers and the Hasura team.

To report a security issue, email us at [build@hasura.io] (mailto: build@hasura.io) Attach all necessary information,
Provide all details.

### When do I need to report a vulnerability?

-You have found a potential security vulnerability Hasura GraphQL Engine or related component.
-Hasura GraphQL Engine that doesn't know how the vulnerability affects it.
-Hasura GraphQL Engine on you think you found the vulnerability in another project (eg Heroku, Docker, etc.).
-Hasura GraphQL Engine users to report other potentially dangerous security risks.

### Should you not report a vulnerability?

-Need help with tuning Hasura GraphQL Engine Security component.
-Need help applying security-related updates.
-The problem is not security related.

### Response to security vulnerabilities

-Each report is approved and analyzed within 3 business days by the project maintainer and security team.
-Reporter keeps updated at each stage Problem analysis and resolution (Triage-> Fix-> Release).

## Public Disclosure Timing

The date of publication is negotiated by the Hasura product security team and its bug submitter. We prefer to fully disclose that bug as soon as possible
Once user mitigation is available. Delaying disclosure is reasonable if the bug or fix is ​​not yet fully understood,
The solution has not been fully tested or for vendor coordination. Disclosure timeframes range from immediate (especially if already published) to weeks.
The period from reporting to public release is normally expected to be around 7 days. The Hasura GraphQL Engine maintainer and security team
Make a final call to set the disclosure date.

(Some sections are inspired and adapted from
[https://github.com/kubernetes/website/blob/master/content/en/docs/reference/issues-
security / security.md] (https://github.com/kubernetes/website/blob/master/content/en/docs/reference/issues-security/security.md))

## Translation
This document is available in the following translated versions:
-[French: fr:] (translations / SECURITY.french.md)

Reply:
Added suggested changes